cli/flags.rs

```diff
@@ -732,6 +733,17 @@ Generate html reports from lcov:
         .help("Output coverage report in lcov format")
         .takes_value(false),
     )
+    .arg(Arg::new("output")
+    .long("output")
```

Comment:
This arg should require `--lcov` arg to be present, otherwise I can do:
```
deno coverage --output=cov.lcov cov/
```
which makes no sense, since it means to print test report to stdout

Reply:
yeah, I somehow missed `requires` method in clap and implemented the following which is obviously not so good, and I will change this now
https://github.com/denoland/deno/blob/eff2796181d20dbe0c8a18dbe45d12c4bb5e57f2/cli/tools/coverage/mod.rs#L618-L630

Reply:
@bartlomieju thanks for the hint,
https://github.com/denoland/deno/blob/c4356c02b47d36411f3f68adc5c43883d4df2d3e/cli/flags.rs#L736-L738